scripts/node/dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM registry.access.redhat.com/ubi8/ubi
+
+RUN yum -y install rpm-build rpm-libs rpm-build-libs rpm
+RUN yum -y install vim-common vim-enhanced vim-filesystem
+
```

Comment:
Can we `COPY`  the `generate-rpm-specs.sh` into the image and get rid of Step 1. in the `README.md`?